config/configtelemetry/doc.go

```diff
@@ -4,4 +4,46 @@
 // Package configtelemetry defines various telemetry level for configuration.
 // It enables every component to have access to telemetry level
 // to enable metrics only when necessary.
+//
+// 1. configtelemetry.None
+//
+// No telemetry data should be collected.
+//
+// 2. configtelemetry.Basic
+//
+// Signals associated with this level cover the essential coverage of the component telemetry.
+//
+// This is the default level recommended when running the collector.
+//
+// The signals associated with this level must have low cardinality.
```

Comment:
Is there a maximum cardinality #  we should recommend here?

Reply:
sure?

Reply:
```suggestion
// The signals associated with this level must show low cardinality, the number of combinations of dimension values. The maximum cardinality allowed is set to 50.
```